src/Tasks/SynTree/Quiz.hs

```diff
@@ -21,6 +24,6 @@ generateSynTreeInst SynTreeConfig {..} = do
       , correct = display tree
       }
 
-feedback :: SynTreeInst -> String -> Bool
+feedback :: SynTreeInst -> PropFormula -> Bool
 feedback SynTreeInst {instSynTree} input =
-  formulaParse input == Right instSynTree
+  formulaToTree input == instSynTree
```

Comment:
This change seems problematic to me. Previously, an input like `A /\ B /\ C` would never be accepted, and that was intended to be so. But now, such inputs could be deemed correct, right?

Relatedly, the following two current tests seem too weak. Previously, they did not only check "parseability", but actually that the correct tree resulted: https://github.com/fmidue/logic-tasks/blob/23d25843046d0dabcf3ee9f7b17a8ef244df7365/test/SynTreeSpec.hs#L73 https://github.com/fmidue/logic-tasks/blob/23d25843046d0dabcf3ee9f7b17a8ef244df7365/test/SynTreeSpec.hs#L76

Reply:
I was not aware that removing redundant brackets was not intended. I'm going to turn this into issues.